src/AppInstallerCLICore/Core.cpp

```diff
@@ -46,11 +46,15 @@ namespace AppInstaller::CLI
     {
         init_apartment();
 
+        Execution::Context context{ std::cout, std::cin };
+        context.EnableCtrlHandler();
+
         // Enable all logging for this phase; we will update once we have the arguments
         Logging::Log().EnableChannel(Logging::Channel::All);
         Logging::Log().SetLevel(Logging::Level::Verbose);
         Logging::AddFileLogger();
-        Logging::EnableWilFailureTelemetry();
+        Logging::Telemetry().EnableWilFailureTelemetry();
```

Comment:
This should be a global thing still, and inside the callback it should be using the thread local Telemetry.